VirtoCommerce.Platform.Web/Licensing/License.cs

```
@@ -86,6 +86,18 @@ private static bool ValidateSignature(string data, string signature)
             return result;
         }
 
+        private static string ReadFileWithKey(string path)
+        {
+            string fileContent;
+
+            using (var streamReader = File.OpenText(path))
```

Need to throw a friendly exception if the public key or license aren't found